prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,69 @@
+from prac_09.taxi import Taxi
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """Program of taxi simulator with choosing a taxi and its distance, showing the bill."""
+    print("Let's drive!")
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    bill_to_date = 0
```

Comment:
Good code, I think I should also use "bill_to_date"! But for now, I use a different name. ^^

Reply:
your code looks good